### PR TITLE
Miscellaneous fixes and improvements for project setup

### PR DESCRIPTION
**Pull request by CypherPoet**
I had some issues setting up, building, and running the app, and I think it’s mainly due to a few inconsistencies and missing values in the `.env.sample` file.

These changes include everything I did to get `HexaConfig` to initialize without crashing — as well as some other additions that I think might help anyone who’s new to the project.

(I should also mention that I’m not entirely certain all of these values are “correct” or ideal 🙂. Any feedback or suggestions would be most welcome.)

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -76,15 +79,18 @@ BIT_API_URLS_MAINNET_BALANCE_CHECK=#endpoint for checking balance against a supp
...
 BIT_API_URLS_MAINNET_TX_LIMIT=#mainnet txn fetch limit specifier (ex: ?limit=100)
 
 # CIPHER_SPEC
 BIT_CIPHER_ALGORITHM=#en/de-cryption algorithm
 BIT_CIPHER_SALT=#en/de-cryption salt
 BIT_CIPHER_KEYLENGTH=#key length for the cipher
 
+
 # SSS Variables
+BIT_KEY_STRETCH_ITERATIONS=# (default: 10000)
```
Comment (CypherPoet) [inline .env.example:92]: I based this on the [latest NIST guidelines](https://pages.nist.gov/800-63-3/sp800-63b.html#sec5) for PBKDF2 iteration count

File: .env.example [hunk 2]
```diff
@@ -112,7 +118,8 @@ BIT_SHARE_HEALTH_TIME_SLOT1=#share slot1 for switching health stages
 BIT_SHARE_HEALTH_TIME_SLOT2=#share slot2 for switching health stages
 
 ## Trusted Contact
-BIT_TC_REQUEST_EXPIRY=#expiration time for TC request in milli-seconds
+BIT_TC_REQUEST_EXPIRY=#expiration time for TC request in milli-seconds (default: 10000)
+BIT_LEGACY_TC_REQUEST_EXPIRY=#expiration time for legacy TC request in milli-seconds (default: 10000)
```
Comment (CypherPoet) [inline .env.example:122]: Not completely sure what the best default is here.

File: .env.example [hunk 3]
```diff
@@ -124,8 +131,13 @@ APP_ID=#app store id
 WALLET_SLUG=#fastbitcoins assigned wallet slug
 FBTC_REGISTRATION_URL=#fast bitcoins registration url
 FBTC_URL=#fastbitcoins api base url
-BIT_FAST_BITCOINS_SERIES=#
-BIT_FAST_BITCOINS_INSTANCE_COUNT=#
+
+BIT_DONATION_ACCOUNT_SERIES= # (default; 100)
+BIT_DONATION_ACCOUNT_INSTANCE_COUNT= # (default; 100)
```
Comment (CypherPoet) [inline .env.example:136]: Not completely sure what the best defaults are here.

File: .env.example [hunk 4]
```diff
@@ -124,8 +131,13 @@ APP_ID=#app store id
...
+
+BIT_FAST_BITCOINS_SERIES= # (default; 100)
+BIT_FAST_BITCOINS_INSTANCE_COUNT= # (default; 100)
```
Comment (CypherPoet) [inline .env.example:139]: Not completely sure what the best defaults are here.

File: .env.example [hunk 5]
```diff
@@ -124,8 +131,13 @@ APP_ID=#app store id
...
+
 ## TRUSTED CONTACTS
-BIT_TRUSTED_CONTACTS_SERIES=#
-BIT_TRUSTED_CONTACTS_INSTANCE_COUNT=#
+BIT_TRUSTED_CONTACTS_SERIES= # (default; 100)
+BIT_TRUSTED_CONTACTS_INSTANCE_COUNT= # (default; 100)
```
Comment (CypherPoet) [inline .env.example:143]: Not completely sure what the best defaults are here.

File: .env.example [hunk 6]
```diff
@@ -124,8 +131,13 @@ APP_ID=#app store id
 WALLET_SLUG=#fastbitcoins assigned wallet slug
 FBTC_REGISTRATION_URL=#fast bitcoins registration url
 FBTC_URL=#fastbitcoins api base url
-BIT_FAST_BITCOINS_SERIES=#
-BIT_FAST_BITCOINS_INSTANCE_COUNT=#
+
+BIT_DONATION_ACCOUNT_SERIES= # (default: 100)
+BIT_DONATION_ACCOUNT_INSTANCE_COUNT= # (default: 100)
```
Comment (CypherPoet) [inline .env.example:136]: Not completely sure what the best defaults are here.

File: .env.example [hunk 7]
```diff
@@ -124,8 +131,13 @@ APP_ID=#app store id
...
+
+BIT_FAST_BITCOINS_SERIES= # (default: 100)
+BIT_FAST_BITCOINS_INSTANCE_COUNT= # (default: 100)
```
Comment (CypherPoet) [inline .env.example:139]: Not completely sure what the best defaults are here.

File: .env.example [hunk 8]
```diff
@@ -124,8 +131,13 @@ APP_ID=#app store id
...
+
 ## TRUSTED CONTACTS
-BIT_TRUSTED_CONTACTS_SERIES=#
-BIT_TRUSTED_CONTACTS_INSTANCE_COUNT=#
+BIT_TRUSTED_CONTACTS_SERIES= # (default: 100)
+BIT_TRUSTED_CONTACTS_INSTANCE_COUNT= # (default: 100)
```
Comment (CypherPoet) [inline .env.example:143]: Not completely sure what the best defaults are here.

**Comment by AliMeer** [comment]
@CypherPoet 
Now that you have the env file. Please re visit your PR and update accordingly.

**Comment by CypherPoet** [comment]
@AliMeer Just updated and rebased

**Comment by AliMeer** [comment]
> @AliMeer Just updated and rebased

Nice improvements, I guess it needed a new perspective to uncover these issues 👍

**Review by AliMeer — APPROVED** [review]
Good Debut! 😊👍

**Comment by CypherPoet** [comment]
@AliMeer Haha, I yep -- I figured I'd take advantage of the process 😄 .

**Pull request merged**